test/grub_install_test.rb

```diff
@@ -92,7 +96,17 @@ def expect_grub2_install(target, device: nil, removable: false)
       it "runs with target arm64-efi on aarch64" do
         stub_arch("aarch64")
         stub_efivars
-        expect_grub2_install("arm64-efi")
+        expect_grub2_install("arm64-efi", removable: true)
+
+        subject.execute(devices: [])
+      end
+
+      it "runs twice as removable and non removable on aarch64 with efi vars (bsc#1167015)" do
+        stub_arch("aarch64")
+        stub_efivars
+        allow(::File).to receive(:directory?).and_return(true)
+        expect_grub2_install("arm64-efi", removable: false)
+        expect_grub2_install("arm64-efi", removable: true)
```

Comment:
Did the sequence matter ? Looks to me the removable should be in front of non-removable ? 
````
expect_grub2_install("arm64-efi", removable: true)
expect_grub2_install("arm64-efi", removable: false)
````

Reply:
no, in this test order does not matter. I can do it in to be dependent on order, but if I get it, order is not important, just calling both.

Reply:
Yes, order doesn't matter as long as we only want both installed. I was misunderstanding the test that it would expect the command execution order to match what's programmed in the source code.